api/views/message.py

```diff
@@ -38,11 +38,24 @@ def message_detail(request, pk):
         """
         We don't really edit messages-- this endpoint updates an associated MessageAudit
         """
-        serialized_audit = MessageAuditSerializer(message.audit, data=request.data)
-        if serialized_audit.is_valid():
-            serialized_audit.save(updated_by=request.user)
-            return Response(serialized_audit.data, status=status.HTTP_201_CREATED)
-        return Response(serialized_audit.errors, status=status.HTTP_400_BAD_REQUEST)
+        if "labels" in request.data:
+            for label in request.data["labels"]:
```

Comment:
Is this for bulk actions later? Would this ever have more than one label?

Reply:
Yeah, I can change that-- it's an artifact of an earlier implementation.